lenskit-eval/src/main/java/org/lenskit/eval/traintest/TrainTestExperiment.java

```diff
@@ -489,15 +466,16 @@ private void buildJobGraph() {
         allJobs = new ArrayList<>();
         TrackedJob tracker = new TrackedJob(JOB_TYPE);
         tracker.getEventBus().register(new StatusLogger(logger));
+        try {
+            tracker.getEventBus().register(new WebAppListener(logger));
+        }catch (Exception e){
+            //Failed to create web app listener
+        }
```

Comment:
This is the change we want to keep. All other changes in this file should be rolled back.